ona_migration_script/migrate_submissions.py

```diff
@@ -0,0 +1,44 @@
+from ona import OnaApiClient
+import settings
+
+FROM_FORM = '15756'  # imaliyethu
+TO_FORM = '23666'  # per report
+
+DEFAULTS = {'msisdn': '', 'fault_status': '', 'logged_date': ''}
+
+client = OnaApiClient(settings.USERNAME, settings.PASSWORD)
```

Comment:
I think everything (including username and password) can just go in args
for now? Fine with argparse too.